synapse/lib/modelrev.py

```diff
@@ -783,6 +784,292 @@ async def revModel_0_2_26(self, layers):
+                                }}
+
+                                if $edges {
+                                    $item = ({
+                                        "node": $iden,
+                                        "view": $rview.iden,
+                                        "direction": "n1",
+                                        "edges": $edges,
+                                    })
+
+                                    $edgeq.put($item)
+                                    $edges = ([])
+                                }
+
+                                // Get N2 edges and store them in the queue
+                                { for $edge in $node.edges(reverse=$lib.true) {
+                                    ($verb, $src) = $edge
+                                    $edges.append($edge)
+
+                                    [ <($verb)- { iden $src } ]
+
+                                    // Flush to the queue if the list grows too large
+                                    if ($edges.size() > 1000) {
+                                        $item = ({
+                                            "node": $iden,
+                                            "view": $rview.iden,
+                                            "direction": "n2",
+                                            "edges": $edges,
+                                        })
+
+                                        $edgeq.put($item)
+                                        $edges = ([])
+                                    }
+                                }}
+
+                                if $edges {
+                                    $item = ({
+                                        "node": $iden,
+                                        "view": $rview.iden,
+                                        "direction": "n2",
+                                        "edges": $edges,
+                                    })
+
+                                    $edgeq.put($item)
+                                    $edges = ([])
+                                }
+                            }
+                        }
+
+                        $item = ({
+                            "node": $iden,
+                            "props": {
+                                "valu": $repr,
+                                "v2_2": $oldcpe.props.v2_2,
+                            },
+                            "view": $view.iden,
+                            "layer": $lib.layer.get().iden,
+                            "tags": $oldcpe.tags(),
+                            "data": $oldcpe.data.list(),
+                            "sources": $sources,
+                        })
+
+                        $cpeq.put($item)
+                    }
+
+                    // Finally, delete the invalid node
+                    $lib.log.debug(`Deleting invalid it:sec:cpe node: {$repr}`)
+                    iden $oldcpe.iden() | delnode --force
```

Comment:
In the unsuccessful case I think you already have the old cpe node in the pipeline

Reply:
OBE.